package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "zoo-modeling-app",
-  "version": "0.27.0",
+  "version": "0.0.0",
```

Comment:
This is on purpose: this field is only changed during nightly or release builds. The version isn't tracked in the codebase anymore with this new process.

Reply:
Ha that's a pretty good idea! Thanks. Instead of simply removing the `make-release.sh` script like now I guess we can just repurpose it to this :) Do you mind if I keep the name then?

Reply:
I'd probably give it a new name like `generate-semantic-release-tag.sh` because `make-release.sh` is not that descriptive and it isn't "making" anything.

Also I'd take my new source code above since I have edited the source of the previous tag and I edited the output so it only `echo` the tag. This means you can directly use the output of the bash script instead of copying and pasting the value from the terminal.

Reply:
Ok cool that works, was thinking that this script could also create the tag and push it but we can for sure keep that manual